packages/kbn-alerts-ui-shared/src/rule_type_modal/components/rule_type_list.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFacetGroup,
+  EuiFacetButton,
+  EuiCard,
+  EuiSpacer,
+  EuiText,
+  EuiEmptyPrompt,
+  EuiButton,
+} from '@elastic/eui';
+import { omit } from 'lodash';
+import { PRODUCER_DISPLAY_NAMES } from '../../common/i18n';
+import { RuleTypeWithDescription, RuleTypeCountsByProducer } from '../types';
+
+interface RuleTypeListProps {
+  ruleTypes: RuleTypeWithDescription[];
+  onSelectRuleType: (ruleTypeId: string) => void;
+  onFilterByProducer: (producer: string | null) => void;
+  selectedProducer: string | null;
+  ruleTypeCountsByProducer: RuleTypeCountsByProducer;
+  onClearFilters: () => void;
+}
+
+const producerToDisplayName = (producer: string) => {
+  return Reflect.get(PRODUCER_DISPLAY_NAMES, producer) ?? producer;
```

Comment:
never seen `Reflect.get` before haha, is this any different than accessing the object normally?